pkg/kubelet/hyper/hyperclient.go

```diff
@@ -343,39 +343,40 @@ func (client *HyperClient) GetPodIDByName(podName string) (string, error) {
 }
 
 func (client *HyperClient) ListPods() ([]HyperPod, error) {
```

Comment:
As issue 136 said, you should never use any types from `pkg/kubelet/hyper/types.go`, you should always return structs defined in `pkg/kubelet/hyper/types/types.pb.go` instead.

And you should also check if the returned struct in  `pkg/kubelet/hyper/types/types.pb.go` is the same in `pkg/kubelet/hyper/types/types.pb.go`, if not, you need to change the code in `pkg/kubelet/hyper/hyper.go` to make it work with structs in `types.pb.go`